src/check_release.rs

```diff
@@ -129,7 +129,7 @@ pub(super) fn run_check_release(
             config.shell_status(
                 "Starting",
                 format_args!(
-                    "{} checks, {} skipped",
+                    "{} checks, {} skipped (allowed by version change)",
```

Comment:
If we are skipping checks because they are allowed, I feel like "ignored" would be a better word than "skipped".

Reply:
Sounds good. To be clear, are you advocating for just `{} ignored` or the more elaborate `{} ignored (allowed by version change)` option?

Reply:
I feel like the parenthetical statement makes it overly long and harder to scan.  Ideally there would be a better way to communicate that

Reply:
A few alternatives:
- `ignored by {} version bump` inserting `major/minor/patch` into the new formatting slot
- `ignored for {} version bump` but otherwise same as above
- `unnecessary` instead of `ignored` in the above two options

Any others you'd suggest?

I agree that ideally we'd have a super-clear one-word way to communicate what's going on. But the current phrasing is unclear and I'm inclined to go for a "better than the status quo" option. We can always polish more in the future if we find a better way to phrase it.

Reply:
Alternatively, we list the reason alongside the check

No strong opinions on what direction is taken atm though

Reply:
Oh that's great feedback — thanks. How about this: I'll swap in `unnecessary` since it seems like a clear and immediate improvement over the status quo, and we can continue finessing the presentation as we do more user research and gather more feedback on what's confusing.

Reply:
> Oh that's great feedback — thanks. How about this: I'll swap in `unnecessary` since it seems like a clear and immediate improvement over the status quo, and we can continue finessing the presentation as we do more user research and gather more feedback on what's confusing.

Sounds great!